CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+# 1.0.10-rc1 (2020-06-23)
+## Edge Agent
+### Features
+* Update codebase to dotnet 3.1. [f87a18a](https://github.com/Azure/iotedge/commit/f87a18a487ea0c05752254aaba04a4f89028120a)
+* Install Trust Bundle in EdgeAgent and EdgeHub. [4f85dcc](https://github.com/Azure/iotedge/commit/4f85dcc7d3fd6d4772d4a9b86ec3ecad651938fd)
+* Enable Metrics with RocksDB [0d2cf76](https://github.com/Azure/iotedge/commit/0d2cf76ea1da561252a1100b7da5b4f0f5f1d471)
```

Comment:
Let's change this to Enable support to collect metrics, or something like that. @veyalla @lfitchett

Reply:
We don't even need to include that, it is internal details.

Reply:
Ok, I'll remove it.